docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -127,39 +127,39 @@ The commands in this section enable you to perform the following actions:
 - Interact with individual rule groups in the Mimir ruler
 - Manipulate local rule files
 
-#### List
+#### List rules
 
 The following command retrieves the names of all rule groups in the Grafana Mimir instance and prints them to the terminal.
 
 ```bash
 mimirtool rules list
 ```
 
-#### Print
+#### Print rules
 
 The following command retrieves all rule groups in the Grafana Mimir instance and prints them to the terminal.
 
 ```bash
 mimirtool rules print
 ```
 
-#### Get
+#### Get rule group
 
 The following command retrieves a single rule group and prints it to the terminal.
 
 ```bash
 mimirtool rules get <namespace> <rule_group_name>
 ```
 
-#### Delete
+#### Delete rule group
 
 The following command deletes a rule group.
 
 ```bash
 mimirtool rules delete <namespace> <rule_group_name>
 ```
 
-#### Load
+#### Load rule groups
```

Comment:
Should this be singular or plural, to provide parallel structure wrt language in subheadings?

Reply:
I would opt for singular tense, and fix all references.

Reply:
It let you load 1 or more rule groups, but I'm fine if you prefer to use singular tense. Going to change it.